tests/components/SearchBar.stories.tsx

```diff
@@ -82,4 +83,5 @@ function conductRecentSearches(textboxEl: HTMLElement) {
   userEvent.clear(textboxEl);
   userEvent.type(textboxEl, 'recent search 2');
   userEvent.keyboard('{enter}');
+  userEvent.clear(textboxEl);
```

Comment:
lets revert the changes in SearchBar stories since it's not behaving as expected with the await approach. We can revisit this later